cpp/client/ct.cc

```diff
@@ -996,6 +997,7 @@ int Monitor() {
 int main(int argc, char** argv) {
   google::SetUsageMessage(argv[0] + string(kUsage));
   util::InitCT(&argc, &argv);
+  ConfigureSerializerForV1CT();
```

Comment:
This is probably necessary to make the tool work, but isn't mentioned in the description.